disperser/apiserver/server.go

```diff
@@ -326,6 +326,15 @@ func (s *DispersalServer) getAccountRate(origin, authenticatedAddress string, qu
 	// Check if the address is in the allowlist
 	if len(authenticatedAddress) > 0 {
 		quorumRates, ok := s.rateConfig.Allowlist[authenticatedAddress]
```

Comment:
how about we always make the keys in this map lowercase and compare with lowercased addresses? 
https://github.com/Layr-Labs/eigenda/blob/master/disperser/apiserver/config.go#L167-L170

Reply:
Thx for pointing this out. That's a way more systemic solution and avoids us having to normalize the allowlist configs.